module1/homework/lengthSort/sort.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+#include <string>
+#include <forward_list>
+#include <deque>
```

Comment:
Order of the includes :)

Reply:
what's wrong?

Reply:
https://coders.school/edu/courses/powtorka-1-qa/lesson/kolejnosc-include/